content/soarian/dstu2/diagnostic/diagnostic-report.md

```diff
@@ -60,7 +60,7 @@ Notes:
 
 #### Request   
 
-    GET https://fhir-myrecord.sandboxcerner.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/DiagnosticReport?patientId=4342008
+    GET <serviceRootUrl>/DiagnosticReport?patientId=1FNHDJ60-D6A1-441F-AA25-FB6FA6719CEA&date=%3E%3D2015-06-01T11%3A55%3A00&date=%3C2015-06-29T11%3A55%3A00
```

Comment:
I think we landed on using an obviously fake tenant id. EG: https://fhir-myrecord.sandboxcerner.com/dstu2/123abc/DiagnosticReport?...

Otherwise, just use /DiagnosticReport?...